lib/spack/llnl/util/cpu/detect.py

```diff
@@ -80,7 +80,7 @@ def check_output(args):
         return subprocess.run(
```

Comment:
Hey, wasn't `subprocess.run` not added until Python 3.5?

Reply:
You're right, [docs here](https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module) for reference.